core-services/image-mirroring/knative/mapping_knative_v0_17_quay

```diff
@@ -0,0 +1,13 @@
+registry.svc.ci.openshift.org/openshift/knative-v0.17.1:knative-eventing-sources-heartbeats-receiver quay.io/openshift-knative/knative-eventing-sources-heartbeats-receiver:v0.17.1
```

Comment:
Curious. Where are these promoted images used? Loking at some older files, we've been only promoting images from Eventing and Client. We haven't promoted images from Serving. So I'm curious whether it makes sense at all.

Reply:
The DP operator does use them, e.g.:  https://github.com/openshift-knative/knative-kafka-operator/blob/master/deploy/resources/knative-eventing-kafka-contrib-v0.16.0.yaml#L325

Reply:
Thanks. Alright.